test/testdata/fixtures/reaper-control-plane/reaper.yaml

```diff
@@ -10,8 +10,6 @@ spec:
     resources:
       requests:
         storage: 256Mi
-  httpManagement:
```

Comment:
This is making Reaper interact with the cluster is repairs via JMX. It's not a problem for this PR because the Control Plane Reaper test uses Cassandra 3.11. It will not work should we ever switch this to DSE and such. Although I don't see why we'd ever do that in the tests here.

Still, I'm wondering, Is there a particular reason you're removing this?

Reply:
Ah, actually.. not in k8ssandra-operator. We made the default to management-api in the other project, not here (yet). Although I think we should consider doing it in the k8ssandra-operator also. So I'll return this back.